packages/matrix-client-server/src/matrixDb/index.ts

```diff
@@ -316,6 +321,122 @@ class MatrixDBmodified implements MatrixDBmodifiedBackend {
     return this.db.updateWithConditions(table, values, conditions)
   }
 
+  // eslint-disable-next-line @typescript-eslint/promise-function-async
+  createOneTimeToken(
```

Comment:
One-time-token means "deleted on read", where is the autoremoval method ?

Reply:
There is a "verifyOneTimeToken" method in the matrix-identity-server db folder, but I did not include it here since I didn't use it. I delete the token manually in the submit token method once I've verified it (since I need to take additional steps to verify the token that aren't in the verifyToken method before deleting it)